src/test/java/lotto/domain/LottoTicketsTest.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import lotto.infrastructure.error.LottoTicketsErrorCode;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class LottoTicketsTest {
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    public void lottoTickets_of_null_or_empty(List<LottoTicket> lottoTickets) {
```

Comment:
@DisplayName 을 통해 어떠한 테스트인지 나타내주면 어떨가요?
작성자가 아닌 사람이 읽었을 경우 테스트에 대한 이해도를 높여줄 수 있을 것 같아요 😄 
```suggestion
    @DisplayName("테스트 설명")
    @ParameterizedTest
    @NullAndEmptySource
    public void lottoTickets_of_null_or_empty(List<LottoTicket> lottoTickets) {
```